float64 rel_angle 			# relative angle to target                        -180 deg        180 deg  
float64 pole_ang_vel 		# pole angular velocity (relative to gripper)  deg/sec?
float64 grip_joint_pos 		# gripper orientation (the joint that moves ) -90 deg         90 deg 
float64 grip_joint_vel		# gripper angular velocity
float64 finger_distance		# distance between two fingers  